simpledotorg/simple-server
# adding new indicators for ethiopia

**Story card:** [https://app.shortcut.com/simpledotorg/story/12468/hypertensive-patients-enrolled-to-care-by-age-and-sex](URL)
https://app.shortcut.com/simpledotorg/story/12432/export-treatment-outcome-for-cohort-of-patient-registered-for-hypertension-treatment-six-months-prior-to-the-reporting-peridod
https://app.shortcut.com/simpledotorg/story/12431/export-total-number-of-cohort-hypertensive-patients-enrolled-to-care-six-month-prior-to-the-reporting-month
https://app.shortcut.com/simpledotorg/story/12468/hypertensive-patients-enrolled-to-care-by-age-and-sex
https://app.shortcut.com/simpledotorg/story/12430/export-hypertensive-patients-by-timing-of-enrollment
https://app.shortcut.com/simpledotorg/story/12433/export-hypertensive-patients-enrolled-to-care-by-type-of-treatment

## Because

There are new indicators required for Ethiopia DHIS2

## This addresses

This export is already behind a feature flag "dhis2_export". Hence, using the same one and not adding any new flags
Removing the old indicators
Adding functionality for new indicators.

## Test instructions

Enable "dhis2_export" feature flag.
Ensure FacilityBusinessIdentifier.identifier_id is linked to the correct facility/organisation id in the DHIS2 that is used for testing.
Set proper credentials in the test environment for the following variables
DHIS2_PASSWORD
DHIS2_URL
DHIS2_USERNAME
DHIS2_DATA_ELEMENTS_FILE

Ensure the file used for DHIS2_DATA_ELEMENTS_FILE has the correct data_element ID mappings
run `rake dhis2:ethiopia_export`

Data should be exported to DHIS2
